### ISSUE TITLE
shebang doesn't recognize additional CLI flags

### ISSUE THREAD

**Issue by pdehaan** [mysticatea/eslint-plugin-node#51]
I'm using **eslint-plugin-node@2.1.1** and my pseudo .eslintrc.yml file is:

```yaml
extends:
  - eslint:recommended
  - plugin:node/recommended

plugins:
  - node

rules:
  node/no-unpublished-bin: error
```

When I run `$ eslint .`, I get the following error:

```sh
> eslint .


/Users/pdehaan/dev/github/mozilla/observatory-cli/index.js
  3:1  error  This file needs shebang "#!/usr/bin/env node"  node/shebang

✖ 1 problem (1 error, 0 warnings)
```

But the first line of my /index.js file is:

```js
#!/usr/bin/env node --harmony
```

It looks like the `node/shebang` rule may be looking for an exact match of `#!/usr/bin/env node` and isn't liking the awkward use of the `--harmony` flag at the end.

Removing the `--harmony` from the shebang in /index.js works as expected.

**Comment by mysticatea** [comment]
Thank you for this issue!

I had not realized that we can give arguments to shebang.
I will fix it within today (JST).

**Comment by mysticatea** [comment]
I published [eslint-plugin-node@2.1.2](https://github.com/mysticatea/eslint-plugin-node/releases/tag/v2.1.2).
